source/developer/fx-guidelines.rst

```diff
@@ -45,7 +45,7 @@ There will be unavoidable product failures, whether through technical issues lik
 
 Features like confirmation dialogs on critical actions or temporarily saving user input in case of disconnect are examples of forgiveness.
 
-That said, forgiving does not mean allowing the product to be used incorrectly, and breaking the obvious principle. As an analogy, if someone kicks a football and misses the goal, being forgiving means giving them a chance to kick again, it does not mean counting the miss as a goal.
+That said, forgiving does not mean allowing the product to be used incorrectly, and breaking the "Obvious" principle. As an analogy, if someone kicks a football and misses the goal, being forgiving means giving them a chance to kick again, it does not mean counting the miss as a goal.
```

Comment:
For readability, should this read: "That said, forgiving does not mean allowing the product to be used incorrectly by breaking the "Obvious" principle."?